tests/utils.py

```diff
@@ -366,3 +369,40 @@ def create_patron(data):
         reindex=True)
     flush_index(PatronsSearch.Meta.index)
     return ptrn
+
+
+def create_user_token(client_name, user, access_token):
+    """Create a token for the given user."""
+    # Create token for user
+    with db.session.begin_nested():
+        c = Client(
```

Comment:
maybe use an explicit name instead of just one character

Reply:
ok, done.